components/IngredientSelector/Navigator.tsx

```diff
@@ -144,12 +187,12 @@ export default function Navigator() {
           -
         </Button>
         <Input
-          type="number"
-          value={quantityValue ?? undefined}
-          onChange={(event) =>
-            dispatch(updateValue({ quantityValue: Number(event.target.value) }))
-          }
-        />
+        type="text"
```

Comment:
Why? number types indicate to the keyboard that it should show numbers. If you want to display extra info, it's better to put it outside.

Joy provides endDecorator for that purpose
https://mui.com/joy-ui/react-input/#decorators

Docs put a select, but it could be a simple text